client/modules/Mobile/MobileSketchView.jsx

```diff
@@ -0,0 +1,200 @@
+/* eslint-disable */
+import React, { useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+import styled from 'styled-components';
+import Header from '../../components/mobile/Header';
+import PreviewFrame from '../IDE/components/PreviewFrame';
+import Screen from '../../components/mobile/MobileScreen';
+import * as ProjectActions from '../IDE/actions/project';
+import * as IDEActions from '../IDE/actions/ide';
+import * as PreferencesActions from '../IDE/actions/preferences';
+import * as ConsoleActions from '../IDE/actions/console';
+import * as FilesActions from '../IDE/actions/files';
+
+import { getHTMLFile } from '../IDE/reducers/files';
+
+
+import { ExitIcon } from '../../common/icons';
+import { remSize } from '../../theme';
+
+
+const Content = styled.div`
+  z-index: 0;
+  margin-top: ${remSize(68)};
+`;
+
+const IconLinkWrapper = styled(Link)`
+  width: 2rem;
+  margin-right: 1.25rem;
+  margin-left: none;
+`;
+
+const MobileSketchView = (props) => {
+  // TODO: useSelector requires react-redux ^7.1.0
+  // const htmlFile = useSelector(state => getHTMLFile(state.files));
+  // const jsFiles = useSelector(state => getJSFiles(state.files));
+  // const cssFiles = useSelector(state => getCSSFiles(state.files));
+  // const files = useSelector(state => state.files);
+
+  const {
+    htmlFile, files, selectedFile, projectName
+  } = props;
+
+  // Actions
+  const {
+    setTextOutput, setGridOutput, setSoundOutput,
+    endSketchRefresh, stopSketch,
+    dispatchConsoleEvent, expandConsole, clearConsole,
+    setBlobUrl,
+  } = props;
+
+  const { preferences, ide } = props;
+
+  useEffect(() => {
+  //   console.log(params);
+  //   getProject(params.project_id, params.username);
+  });
+
+  return (
+    <Screen>
+      <Header>
+        <IconLinkWrapper to="/mobile" aria-label="Return to original editor">
+          <ExitIcon viewBox="0 0 16 16" />
+        </IconLinkWrapper>
+        <div>
+          <h2>{projectName}</h2>
+          <h3><br /></h3>
```

Comment:
What's the purpose of this mark-up?

`<h3><br /></h3>`

Reply:
It just heightens the header by an `h3` height, to keep the `h2` tag aligned with the previous screen - couldn't think of a better way to do this...

Reply:
We might have to use CSS to add padding to the h2? And maybe set the previous screens header to a know height?

Using `h3` is this way is not semantic especially as screen-readers allow the user to list and navigate via the headings in the page, which will be empty.

Reply:
Good point. I'm fixing this on the next PR, and improving the `<Header />` component as well